rtengine/procparams.cc

```diff
@@ -129,22 +131,22 @@ RetinexParams::RetinexParams ()
     setDefaults ();
 }
 
-void RetinexParams::getDefaulttransmissionCurve(std::vector<double> &curve)
+void RetinexParams::getDefaulttransmissionCurve (std::vector<double> &curve)
```

Comment:
@Beep6581 Our `astyle` rules don't change that aspect (space or not in front of a function call or function declaration(definition).